verification/curator-service/ui/src/components/new-case-form-fields/CaseFormValues.tsx

```diff
@@ -36,6 +36,10 @@ export default interface CaseFormValues {
     notes: string;
 }
 
+export interface CaseReferenceForm extends CaseReference {
```

Comment:
Should we define/export this in the Source component, rather than creating this interface in both forms?

Reply:
Good idea, done.